docs/content.md

```diff
@@ -555,6 +556,44 @@ Alternatively, if you need to insert a mailing lists component in an erb file, y
 )%>
 ```
 
+### Green arrow link (CTA) component
+
+You can use the green arrow link (call to action) component to create a more prominent hyperlink call to action. You can use the component:
+
+* when it's the next logical step on users' journeys
+* more than once on a page
+
+The component should not:
+
+* be used on every page, if there isn't a suitable use for it
+* to direct to pages that are not useful for most people e.g. veterans, SKITT
+* to direct to external sites like UCAS or Teach First
+
+It takes the following parameters and both should be specified:
+
+* link_text
+* link_target
+
+```yaml
+---
+cta_arrow_link:
+  arrowtime:
```

Comment:
I would call this something more self explanatory that demonstrates this is only a placeholder value and can take any name the user wishes